ui/src/components/widgets/Status.vue

```diff
@@ -166,23 +166,24 @@ export default {
       if (!(state && this.displayText)) {
         return ''
       }
```

Comment:
```sugestion
        result = ''
```
???

Reply:
Trying to retain the same behavior of widget. Suggested implementation will make the conditions in L182 a little complicated and hard to understand.